### ISSUE TITLE
Configuration - Comments/Questions

### ISSUE THREAD

**Issue by gsmet** [redhat-developer-demos/quarkus-tutorial#150]
In https://redhat-developer-demos.github.io/quarkus-tutorial/quarkus-tutorial/03_configuration.html ,

- I wonder if at the end of `Adding a config property to application.properties`, we should have a small sentence about live reload?
- I think the note `If you’re still running in live testing mode you will already see that the tests pass again and you can ignore the rest of this page and move on to the next step.` should be made more prominent. And maybe also point to where you can see the Continuous Testing status (either in Dev UI or in the console)

I also wonder why we are talking about disabling continuous testing so soon? Because it kinda breaks the nice promise of never quitting live reload mode.

**Comment by kdubois** [comment]
I agree, the tutorial should be updated to just run the tests with continuous testing & not interrupt the flow with killing dev mode, running the tests manually and then starting it up again.
